### DISCUSSION TITLE
Killing notes

### DISCUSSION THREAD

**Discussion by AltronMaxX** [KadeArchive/Kade-Engine#2638] [Q&A]
Hi, does anyone know how to add killing notes, like from tricky mod, to Kade Engine 1.8?

**Comment by TheLeerName** [comment]
[on channel of bbpanzu](https://www.youtube.com/watch?v=f3ul6Vi029w) there is a tutorial on adding custom notes

**Reply by AltronMaxX** [reply]
This tutorial for andromeda engine :)

**Comment by AltronMaxX** [comment]
Small addition. I want to understand how can I make tricky and normal notes at same time. My code can give me only one type of notes in game ( .
